packages/plugin-e2e/docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: "3.7"
```

Comment:
I'm likely missing something here but I'm wondering if we should ship docker-compose files with this package. It might be to nicer to provide documentation explaining that to run e2e tests the dev needs to build their plugin, provision some dashboards/datasources and start the Grafana server (however they please).

Reply:
In the next PR, I'm going to add some playwright tests also to this repo. The tests (and the docker-compose file) won't be a part of the npm package, but they will be really useful while developing the package since you then don't have to `yarn link` a plugin. Sorry should have explained this in the PR description

Reply:
Ah ok, thanks for the explanation. Let's keep it as it sounds like it'll be very useful later on.